ios/Sources/MediaFeature/MediaListFeature.swift

```diff
@@ -71,8 +72,16 @@ let mediaListReducer = Reducer<MediaListState, MediaListAction, MediaEnvironment
     case .moreDismissed:
         state.moreActiveType = nil
         return .none
-    case .tap(let content):
-        // TODO: open content page
+    case .tap(let id):
+        if let selectedFeedContent = state.feedContents.first(where: { $0.id == id }),
```

Comment:
if we weed to search the feedContent, I think it's better to path the whole `FeedContent` or path the url, not id.